src/xdocs/config_coding.xml

```diff
@@ -4294,7 +4294,24 @@ for(int i=0; i&lt;10; i++) {
         <source>
 &lt;module name=&quot;NestedIfDepth&quot;/&gt;
         </source>
-
+        <p>Valid Code Example:</p>
+        <source>
+if (x &gt; 2) {
+    if (x &gt; 3) {}
+    else {} // OK
+}
+        </source>
+        <p>Invalid Code Example:</p>
+        <source>
+if (x &gt; 2) {
```

Comment:
instead of writting like this: 
```

if (x &gt; 2) {
    if (x &gt; 3) {}
    else {} // OK
}
```
Is it looking good to write like this: 
```

if (true) {
    if (true) {}
    else {} // OK
}
```

Reply:
done.

Reply:
resolved